source/simulator/parameters.cc

```diff
@@ -739,6 +739,28 @@ namespace aspect
       prm.declare_entry ("Names of fields", "",
                          Patterns::List(Patterns::Anything()),
                          "A user-defined name for each of the compositional fields requested.");
+      prm.declare_entry ("Compositional field methods", "",
+                         Patterns::List (Patterns::Selection("continuous field|particles")),
+                         "A comma separated list denoting the solution method of each "
+                         "compositional field. Each entry of the list must be "
+                         "one of the currently implemented field types: "
+                         "``continuous_fem_field'', ``discontinuous_fem_field'', "
+                         "or ``particles''.");
+      prm.declare_entry ("Mapped particle properties", "",
+                         Patterns::Map (Patterns::Anything(),
+                                        Patterns::Anything()),
+                         "A comma separated list denoting the particle properties "
```

Comment:
If it is a comma separated list, shouldn't the pattern then be a `Patterns::List(Patterns::Map(...), ',')`?